store/sqlstore/upgrade.go

```diff
@@ -1238,7 +1243,7 @@ func fixCRTThreadCountsAndUnreads(sqlStore *SqlStore) {
 	`
 	updateThreadMembershipQuery := `
 		WITH q as (` + threadMembershipsCTE + `)
-		UPDATE ThreadMemberships set LastViewed = q.CM_LastViewedAt, UnreadMentions = 0, LastUpdated = :Now
+		UPDATE ThreadMemberships set LastViewed = q.CM_LastViewedAt + 1, UnreadMentions = 0, LastUpdated = :Now
```

Comment:
Bump LastViewed count by 1 so that threads don't marked as unread. This is not strictly necessary after the fixes in the thread_store, but just in case 🤞